src/ast/passes/codegen_llvm.h

```diff
@@ -230,6 +241,7 @@ class CodegenLLVM : public Visitor
   int next_probe_index_ = 1;
   // Used if there are duplicate USDT entries
   int current_usdt_location_index_{ 0 };
+  bool inside_subprog_;
```

Comment:
nit: just to make it explicit
```suggestion
  bool inside_subprog_ = false;
```